frontend/src/components/VCSProviderBasicInfoPanel.vue

```diff
@@ -20,6 +21,19 @@
         }}
       </label>
     </div>
+    <div v-if="isDev" class="radio space-x-2">
+      <input
+        v-model="config.type"
+        name="GitHub.com"
+        tabindex="-1"
+        type="radio"
+        class="btn"
+        value="GITHUB_COM"
+        @change="changeType()"
+      />
+      <img class="h-6 w-auto" src="../assets/github-logo.svg" />
+      <label class="whitespace-nowrap">GitHub.com</label>
+    </div>
```

Comment:
I think it is enough to only guard UI behind the `isDev` flag here since it prevents from having a GitHub Git provider entirely.